lib/levels.js

```diff
@@ -119,7 +120,19 @@ function getLevel (level) {
 function isLevelEnabled (logLevel) {
   const { values } = this.levels
   const logLevelVal = values[logLevel]
-  return logLevelVal !== undefined && (logLevelVal >= this[levelValSym])
+  return logLevelVal !== undefined && this[levelCompSym](logLevelVal, this[levelValSym])
+}
+
+function genLevelComparison (levelComparison) {
+  if (levelComparison === 'ASC') {
+    return (current, expected) => current >= expected
+  }
+
+  if (levelComparison === 'DESC') {
+    return (current, expected) => current <= expected
+  }
```

Comment:
Can you please avoid closures and define top level functions?

Reply:
Sure, done.